src/applications/hca/components/ConfirmationPage/ConfirmationPrintView.jsx

```diff
@@ -34,14 +34,16 @@ const ConfirmationPrintView = ({ name, timestamp }) => {
           <dt className="vads-u-font-family--serif vads-u-font-weight--bold">
             Veteran’s name
           </dt>
-          <dd className="hca-veteran-fullname">{name}</dd>
+          <dd className="hca-veteran-fullname dd-privacy-mask">{name}</dd>
```

Comment:
In some recent work, I had to add both the `dd-privacy-{mask/hidden}` class AND a `data-dd-action-name` to every bit of PII because clicking on the element would show the PII in the event in the side panel.